services/ui-src/src/components/reports/StandardReportPage.tsx

```diff
@@ -58,6 +59,42 @@ export const StandardReportPage = ({ route, validateOnRender }: Props) => {
     navigate(nextRoute);
   };
 
+  const filterResubmissionData = (formData: FormJson) => {
+    //make deep copies of formData
+    const formDataCopy = JSON.parse(JSON.stringify(formData));
+    const formDataFields = JSON.parse(JSON.stringify(formData?.fields));
```

Comment:
suggestion: there's a native js function for this now called `structuredClone()`! does basically the same thing as json.parse(json.stringify)

Reply:
oh nice! Thanks